scripts/devenv-builder/configure-vm.sh

```diff
@@ -28,6 +28,7 @@ function usage() {
     echo "  --skip-dnf-update         Do NOT run dnf update"
     echo "  --pull-images             Force pulling of container images before starting MicroShift"
     echo "  --optional-rpms           Install optional RPMs (OLM, Multus) along with core MicroShift RPMs"
+    echo "  --optional-skipped-rpms   Skip specific optional RPMs list, expects a RPMs list separated by comma."
```

Comment:
Nit: make it consistent with `--skip-dnf-update`?
```suggestion
    echo "  --skip-optional-rpms   Comma separated list of optional RPMs to skip"
```